doc-label-check/.github/workflows/ci.yml

```diff
@@ -0,0 +1,54 @@
+name: CI
```

Comment:
Do we still need `.github` here?

Reply:
Thanks. I have deleted all unnecessary files.